lang/en/local_smartmedia.php

```diff
@@ -44,6 +44,12 @@
 $string['provision:s3usersecretkey'] = 'Smart media S3 user secret key: {$a}';
 $string['provision:stackcreated'] = 'Cloudformation stack created. Stack ID is: {$a}';
 $string['provision:uploadlambdaarchives'] = 'Uploading Lambda function archives to resource S3 bucket';
+$string['report:duration'] = 'Duration (sec)';
+$string['report:filename'] = 'File';
+$string['report:resolution'] = 'Resolution';
+$string['report:size'] = 'Size (Mb)';
+$string['report:transcodecost'] = 'Transcode Cost (AUD)';
```

Comment:
This will change currency based on the configured AWS region.
I'd just remove the AUD for now. Bonus points in the future would be to update this based on the currency region.

Reply:
Removed currecy, TODO will be to add a regional feature got calculation